CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+[official communication channels](https://xournalpp.github.io/community/help/).
+
+## Contributing translations
+
+If you would like to contribute translations, you can submit improvements to
+[our project on Crowdin](https://crowdin.com/project/xournalpp).
+
+## Contributing code changes with pull requests (PRs)
+
+To make a code change, you'll need to first fork the repository on GitHub, if
+you haven't already. To do so, click the "Fork" button at the top right corner
+of the repository's web page.
+
+We assume that you are already familiar with Git, which we use to track changes.
+An excellent resource for Git is the _Pro Git_ book by Scott Chacon and Ben
+Straub, which is available online for free
+[here](https://git-scm.com/book/en/v2).
+
+You should try to first clone your fork and then compile it manually. The
+instructions for compiling Xournal++ on your operating system can be found at
+[`LinuxBuild.md`](readme/LinuxBuild.md), [`MacBuild.md`](readme/MacBuild.md),
+and [`WindowsBuild.md`](readme/WindowsBuild.md). Once you are set up, you will
+be ready to make code changes.
+
+### Sending your contributions for review
+
+When you feel like your are ready to submit your code changes to be integrated
+into the main Xournal++ repository, you will want to submit a pull request (PR).
+To do so, push your changes on to a branch of your fork in GitHub, and then hit
+the "New pull request" button on the [main repository's PR
+tracker](https://github.com/xournalpp/xournalpp/pulls). Select the branch you
+want to contribute, and then press "Create pull request." If you feel like your
+work isn't quite ready, feel free to submit your PR as a draft.
+
+In your description, please include any relevant issue numbers that your PR will
+fix, as well as a short description of what your changes are and why are you
+making them. Feel free to also include questions or comments about code snippets
+you may be unsure about.
+
+### The review process
+
+To maintain the quality of contributions, we typically require a PR to meet the
+following requirements before it is merged.
+
+* **The PR is reviewed by at least one maintainer**, and the relevant feedback is
+  addressed.
+* The code **successfully compiles** and passes all tests on our automated testing
+  infrastructure.
+* The code **passes the automated formatting check**. If the code does not pass,
+  you can copy and paste the required changes from the Azure Pipelines page that
+  has details on how the check failed.
+* Before the PR is merged, a grace period of at least 24 hours have passed since
+the last maintainer has approved the PR. This allows time for objections or
+last-minute feedback. Larger or more complicated PRs may take a longer grace
+period.
+
+Once a PR is ready to merge, we typically ask submitters to rebase or squash
+their commits in order to keep the change history cleaner. If you do not know
+how to do this, feel free to either ask for help.
```

Comment:
This last sentence doesn't look complete to me. Is there an alternative to asking for help?

Reply:
Initially I wanted to say you could either 1) ask how to do it or 2) ask the maintainer to do it for you, but I changed it to just "asking for help" to simplicity. Just forgot to remove "either."